native/react-native/common-components/modals/confirm-modal.js

```diff
@@ -0,0 +1,182 @@
+import React, { Component } from "react";
+import { Modal, Text, TouchableHighlight, View, StyleSheet } from "react-native";
+import { connect } from "react-redux";
+import { PropTypes } from "prop-types";
+import { configureConfirmModal } from "../../redux"; //This action creator is included in this folder ./modalActions.js
+
+const mapStateToProps = state => ({
+    modalState: state.modalState
+});
+
+/**
+ * Basic shape of state.modalState (referenced above in mapStateToProps), which should go in your reducer for this Redux property:
+ *    confirmModalState: {
+        isConfirmModalVisible: false,
+        titleMessage: "",
+        bodyMessage: "",
+        confirmFunction: null,
+        confirmMessage: "",
+        cancelMessage: ""
+    };
+ */
+
+const mapDispatchToProps = dispatch => ({
+    configureConfirmModal: (modalStateObject) => dispatch(configureConfirmModal(modalStateObject))
+});
+
+//We use this to reset the modal when we're done with it.
+const defaultModalState = {
+    isConfirmModalVisible: false,
+    titleMessage: "",
+    bodyMessage: "",
+    confirmFunction: null,
+    confirmMessage: "",
+    cancelMessage: ""
+};
+
+const styles = StyleSheet.create({
```

Comment:
Why include styles alone? What does it look like completely unstyled, are these fonts on both android and ios? etc.